.github/ISSUE_TEMPLATE.md

```diff
@@ -12,7 +13,7 @@ EXPLAIN WHAT YOU DID, PREFERABLY WITH CODE EXAMPLES, HERE.
 ### What happened
 EXPLAIN WHAT IS ACTUALLY HAPPENING, HERE.
 
-### Expected Behavior
+### What you expected to happen
```

Comment:
Nope: "What you did" -> "What happened" -> "What you expected to happen" makes sense to me.